### PR TITLE
media-video/rtmpdump: Move to EAPI6 and git-r3.

### PR DESCRIPTION
**Pull request by bjacquin**
Specify dev-libs/openssl and dev-libs/libressl SLOT.

Package-Manager: portage-2.2.28

### CODE REVIEW

**Comment by bjacquin** [comment]
Follow up from https://github.com/gentoo/gentoo/pull/2210

**Comment by gktrk** [comment]
@hwoarang

**Comment by hwoarang** [comment]
So besides moving it to EAPI6 and git-r3, you also add a new ebuild. You should mention that in the commit message

**Comment by monsieurp** [comment]
ping

**Comment by bjacquin** [comment]
EAPI bump require an ebuild revision bump as well, hence this new ebuild. I just amended commit message to give that information.

File: media-video/rtmpdump/rtmpdump-9999.ebuild [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI="4"
+EAPI="6"
 
-inherit git-2 multilib toolchain-funcs multilib-minimal flag-o-matic
+inherit git-r3 multilib toolchain-funcs multilib-minimal flag-o-matic
```
Comment (mgorny) [inline media-video/rtmpdump/rtmpdump-9999.ebuild]: multilib isn't used (directly) anymore.

File: media-video/rtmpdump/rtmpdump-9999.ebuild [hunk 2]
```diff
@@ -24,6 +24,9 @@ DEPEND="ssl? (
 	)"
 RDEPEND="${DEPEND}"
 
+DOCS=( README ChangeLog )
+HTML_DOCS=( rtmpdump.1.html rtmpgw.8.html )
```
Comment (mgorny) [inline media-video/rtmpdump/rtmpdump-9999.ebuild]: Strictly speaking, I'm not sure if there's a point in installing HTML versions of man pages. We already install man pages, don't we?

File: media-video/rtmpdump/rtmpdump-9999.ebuild [hunk 3]
```diff
@@ -24,6 +24,9 @@ DEPEND="ssl? (
 	)"
 RDEPEND="${DEPEND}"
 
+DOCS=( README ChangeLog )
```
Comment (mgorny) [inline media-video/rtmpdump/rtmpdump-9999.ebuild]: Looks to be covered by the default. Are we disabling some files here?

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		OPT="${CFLAGS}" XLDFLAGS="${LDFLAGS}" CRYPTO="${crypto}" SYS=posix
+}
+
+multilib_src_install() {
+	mkdir -p "${ED}"/usr/$(get_libdir) || die
+	if ! multilib_is_native_abi; then
+		cd librtmp || die
+	fi
+	emake DESTDIR="${ED}" prefix="/usr" mandir="/usr/share/man" \
+	CRYPTO="${crypto}" install
```
Comment (mgorny) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: Missing indent.

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+HTML_DOCS=( rtmpdump.1.html rtmpgw.8.html )
+
+pkg_setup() {
+	if ! use ssl && { use gnutls || use polarssl; }; then
+		ewarn "USE='gnutls polarssl' are ignored without USE='ssl'."
+		ewarn "Please review the local USE flags for this package."
+	fi
+}
+
+src_unpack() {
```
Comment (mgorny) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: This doesn't really look like a clean EAPI 6 solution to me. Lemme guess… S=${WORKDIR} breaks multilib-minimal?

File: media-video/rtmpdump/rtmpdump-9999.ebuild (hunk 2 above)
Comment (bjacquin) [inline media-video/rtmpdump/rtmpdump-9999.ebuild]: We are, dropping it

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild (hunk 5 above)
Comment (bjacquin) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: This is right :"/

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild (hunk 4 above)
Comment (bjacquin) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: Fixed in last PR. Thanks!

File: media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild (hunk 5 above)
Comment (hwoarang) [inline media-video/rtmpdump/rtmpdump-2.4_p20131018-r1.ebuild]: That could be arranged

**Comment by mgorny** [comment]
@hwoarang, ping. What's the status on this one?

**Comment by mgorny** [comment]
Actually, it seems that the EAPI=6 changes has been made as part of 2.4_p20161210. I see that 9999 is outdated, so feel free to create a new PR syncing it to 20161210.

**Pull request closed without merging**